packages/ffe-buttons/less/button-group.less

```diff
@@ -14,12 +14,12 @@
 //
 // Styleguide ffe-buttons.button-group
 .ffe-button-group {
-    display: inline-flex;
+    display: flex;
```

Comment:
Changing direction isn't enough? This should be OK since it's designed to use 100% width on small screens, but the reason we used `inline-flex` was #173

Reply:
`inline-flex` makes the container an inline element, which in turn allows the groups themselves to stack horizontally if the buttons are narrow enough.
![image](https://user-images.githubusercontent.com/435037/38986770-af156990-43cd-11e8-8736-a562f490d58c.png)

Reply:
Tested the patch locally and it does not reintroduce #173, so we're good to go :rocket: